test/ParseableInterface/stored-properties-client.swift

```diff
@@ -0,0 +1,144 @@
+// RUN: %empty-directory(%t)
+
+// RUN: %target-swift-frontend -typecheck %S/stored-properties.swift -module-name StoredProperties -emit-parseable-module-interface-path %t/StoredProperties.swiftinterface
+// RUN: %target-swift-frontend -emit-ir %s -I %t -enable-parseable-module-interface | %FileCheck %s -check-prefix CHECK -check-prefix COMMON
+
+// RUN: %target-swift-frontend -typecheck %S/stored-properties.swift -enable-resilience -module-name StoredProperties -emit-parseable-module-interface-path %t/StoredProperties.swiftinterface
+// RUN: %target-swift-frontend -emit-ir %s -I %t -enable-parseable-module-interface | %FileCheck %s -check-prefix RESILIENT -implicit-check-not CHECK -check-prefix COMMON
```

Comment:
`-implicit-check-not` adds a negative pattern, not a negative prefix. What do you actually want to check here?

Reply:
I don't think there's a way to do that today, but I think it's okay for this test.